src/pages/PlanPage/subRoutes/UpgradePlanPage/UpgradeDetails/UpgradeDetails.tsx

```diff
@@ -1,17 +1,13 @@
 import { IndividualPlan } from 'services/account'
-import { isSentryPlan, Plans } from 'shared/utils/billing'
 
 import ProPlanDetails from './ProPlanDetails'
 import SentryPlanDetails from './SentryPlanDetails'
 import TeamPlanDetails from './TeamPlanDetails'
 
 function UpgradeDetails({ selectedPlan }: { selectedPlan: IndividualPlan }) {
-  if (isSentryPlan(selectedPlan.value)) {
+  if (selectedPlan.isSentryPlan) {
```

Comment:
This is why I wanted to lump in the updates to IndividualPlan (and useAvailablePlans) in this PR, so we didn't have to do a similar short term bandage like we did on lines 12-14 previously.